Description: Kernel 3.18.41.9 Release 
 .
 kali-gemini-linux (3.18.41.9-1) kali-rolling; urgency=medium
 .
   * Release kernel 3.18.41.9
Author: Carsten Boeving <carsten.boeving@whitedome.com.au>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-10-22

--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.alias
@@ -0,0 +1,720 @@
+# Aliases extracted from modules themselves.
+alias usb:v0BDApA811d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v056Ep4007d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0411p029Bd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0846p9052d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2019pAB32d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0411p0242d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v056Ep400Fd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v056Ep400Ed*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0E66p0023d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3318d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3314d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v04BBp0953d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v7392pA813d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v7392pA812d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v7392pA811d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp0823d*dc*dsc*dp*icFFiscFFipFFin* 8812au
+alias usb:v0BDAp0820d*dc*dsc*dp*icFFiscFFipFFin* 8812au
+alias usb:v0BDApA811d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp8822d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp0821d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp0811d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp881Ad*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2604p0012d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp8812d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v148Fp9097d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v050Dp1109d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0411p025Dd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v20F4p805Bd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p0122d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p010Fd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p010Ed*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p010Dd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p0103d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2357p0101d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v13B1p003Fd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3316d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3315d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3313d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p330Ed*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0846p9051d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v07B8p8812d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2019pAB30d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v1740p0100d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v1058p0632d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v2001p3313d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0586p3426d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0E66p0022d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0B05p17D2d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0409p0408d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0789p016Ed*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v04BBp0952d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0DF6p0074d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v7392pA822d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v050Dp1106d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp881Cd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp881Bd*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp881Ad*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v0BDAp8812d*dc*dsc*dp*ic*isc*ip*in* 8812au
+alias usb:v1737p0073d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v1B75p8187d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v18E8p6232d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v13D1pABE6d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v1371p9401d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v114Bp0150d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0DF6p0029d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0DF6p0028d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0DF6p000Dd*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v03F0pCA02d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0846p4260d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0846p6A00d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0846p6100d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0789p010Cd*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0769p11F2d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0BDAp8198d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0BDAp8197d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0BDAp8189d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0BDAp8187d*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v050Dp705Ed*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v0B05p171Dd*dc*dsc*dp*ic*isc*ip*in* rtl8187
+alias usb:v7392p7822d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2357p0100d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v20F4p624Dd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019pAB2Bd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2001p330Dd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2001p330Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2001p3309d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2001p3307d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0E66p0019d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0DF6p0061d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8186d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0B05p17ABd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0846pF001d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0846p9021d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v07B8p8178d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v07AAp0056d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0586p341Fd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v050Dp2103d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v050Dp2102d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v050Dp1004d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019p1201d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFFCd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFFBd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFF8d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFFAd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFF9d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v04F2pAFF7d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp317Fd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v9846p9041d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v4855p0091d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v4855p0090d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v13D3p3359d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v13D3p3358d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v7392p7811d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v20F4p648Bd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019pED17d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019pAB2Ed*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019pAB2Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2019p4902d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v2001p3308d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v13D3p3357d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v103Cp1629d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v4856p0091d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0EB0p9071d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0DF6p0077d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0DF6p0070d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0DF6p005Cd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0DF6p0052d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp5088d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0B05p17BAd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0846p9041d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v07B8p8189d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v07B8p8188d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v06F8pE033d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v050Dp11F2d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v050Dp1102d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Cd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8178d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8754d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp819Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp818Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Fd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Ed*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Dd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Bd*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp817Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8177d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8176d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8170d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp018Ad*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v0BDAp8191d*dc*dsc*dp*ic*isc*ip*in* rtl8192cu
+alias usb:v5A57p0260d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0F88p3012d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0EB0p9020d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0769p11F3d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v114Bp0110d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0707pEE13d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0681p3C06d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v079Bp004Bd*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v148Fp9020d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v148Fp2570d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v148Fp1706d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0DB0p6869d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0DB0p6865d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0DB0p6861d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0411p0097d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0411p008Bd*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0411p0067d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0411p0066d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0411p005Ed*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v06F8pE000d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v1044p8007d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v1044p8001d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v2001p3C00d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v14B2p3C02d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v13B1p001Ad*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v13B1p0011d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v13B1p000Dd*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v050Dp7051d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v050Dp7050d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0B05p1707d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0B05p1706d*dc*dsc*dp*ic*isc*ip*in* rt2500usb
+alias usb:v0586p3415d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0CDEp001Cd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v7167p3840d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v2019pAB50d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v2019pAB01d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0471p200Ad*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v6933p5001d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0769p31F3d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DF6p9712d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DF6p90ACd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DF6p002Fd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DF6p0027d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DF6p0024d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1740p7100d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v04E8p4471d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v18E8p6238d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v18E8p6229d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v18E8p6196d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0812p3101d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v148Fp2671d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v148Fp2573d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v04BBp093Dd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1B75p7318d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DB0pA874d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DB0pA861d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DB0p6874d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DB0p6877d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0DB0p4600d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v13B1p0028d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v13B1p0023d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v13B1p0020d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v06F8pE020d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v06F8pE010d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v06F8pE002d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1472p0009d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1044p800Ad*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1044p8008d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v15A9p0004d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1740p3701d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v7392p7618d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v7392p7318d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07D1p3C07d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07D1p3C06d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07D1p3C04d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07D1p3C03d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07AAp002Ed*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v14B2p3C22d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1371p9032d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1371p9022d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v178Dp02BEd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p0137d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p0119d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p0116d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p00F4d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p00E6d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p00D9d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0411p00D8d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v08DDp0120d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1631pC019d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v050Dp905Cd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v050Dp905Bd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v050Dp705Ad*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v050Dp7050d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0B05p1724d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0B05p1723d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v1690p0722d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v18C5p0002d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v0EB0p9021d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v148Fp9021d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v14B2p3C10d*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07B8pB21Fd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07B8pB21Ed*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07B8pB21Dd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07B8pB21Cd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:v07B8pB21Bd*dc*dsc*dp*ic*isc*ip*in* rt73usb
+alias usb:vF201p5370d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0254d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApF511d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApD522d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApC522d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApA512d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0078d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p006Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0069d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0053d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p004Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p004Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p003Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v08B9p1197d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v18E8p6259d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019pAB29d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019pAB24d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1D4Dp0010d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v05A6p0101d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v100Dp9032d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0169d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0168d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p0615d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p0605d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp094Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148FpF101d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v06F8pE036d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1044p800Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v15A9p0010d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p0602d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p0600d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v203Dp14A1d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C0Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v18C5p0008d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07AAp0042d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07AAp0041d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p0150d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p0148d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p012Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp1003d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3401d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3400d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3399d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3340d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3322d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3284d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3262d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p17A7d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1790d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1761d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1760d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p166Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E0Bp9041d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E0Bp9031d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C11d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C08d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p3074d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p3073d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v20F4p724Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp5572d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A13d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C21d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p0253d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p0241d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v057Cp8501d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A32d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp5372d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp5370d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p2182d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p2181d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p2180d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p2126d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p2104d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04DAp23F6d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04DAp1800d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04DAp1801d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A42d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A22d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C23d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C22d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C20d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C19d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C15d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3365d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3329d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p17E8d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A32d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v043Ep7A12d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p3421d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p006Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p006Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0067d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3573d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019pED19d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0846p9019d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0846p9013d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0846p9012d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p016Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13B1p003Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp094Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0021d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0020d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p7733d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148FpF301d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp1103d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p17ADd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p17BCd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1B75p7733d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v5A57p0284d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0930p0A07d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0068d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0066d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0065d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0062d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0041d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3572d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0170d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1737p0079d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13B1p002Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp0944d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9801d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v167Bp4001d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p179Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1690p0764d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1690p0761d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1690p0744d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p3572d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0165d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0163d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0050d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp8070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3370d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p20DDd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApB511d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C17d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp945Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p343Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p341Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p341Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p3418d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0586p3416d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v5A57p5257d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v5A57p0283d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v5A57p0282d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v5A57p0280d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0CDEp0025d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0CDEp0022d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v157Ep3013d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v157Ep300Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0324d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0323d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0313d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0302d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0164d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v177Fp0153d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v15A9p0006d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApB522d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApA703d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApA702d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApA701d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083ApA618d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083Ap8522d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083Ap7522d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083Ap7512d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083Ap7511d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v083Ap6618d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0060d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p005Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0051d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0048d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0047d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0042d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0040d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p003Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p003Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p003Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p003Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0039d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p002Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p002Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p002Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DF6p0017d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v129Bp1828d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04E8p2018d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3072d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3071d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp3070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp2870d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp2770d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v148Fp2070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1A32p0304d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019pED06d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019pAB25d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2019p5201d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0471p200Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1D4Dp0011d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1D4Dp000Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1D4Dp000Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1D4Dp0002d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v20B8p8888d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1B75pA200d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1B75p3072d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1B75p3071d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p899Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p871Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p871Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p871Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p870Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p822Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p822Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p822Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p821Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p6899d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p3871d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p3870d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p3822d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p3821d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0DB0p3820d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v100Dp9031d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0166d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0164d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0163d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0789p0162d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1737p0078d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1737p0077d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1737p0071d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1737p0070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13B1p0031d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp0948d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp0947d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v04BBp0945d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0018d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0017d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0013d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p000Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0009d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0003d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0E66p0001d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1044p800Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1044p800Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v15A9p0012d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9709d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9708d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9707d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9706d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9705d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9703d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9702d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1740p9701d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v203Dp14A9d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v203Dp1480d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p7722d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p7718d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p7717d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p7711d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v7392p4085d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0FE9pB307d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07FAp7712d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C25d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3C1Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v2001p3317d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C17d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C16d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C15d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C13d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C11d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C0Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C0Ed*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C0Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C0Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07D1p3C09d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v18C5p0012d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07AAp003Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07AAp003Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07AAp002Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p01A8d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p01EEd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p01A2d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p016Fd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p015Dd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p0158d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0411p00E8d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp935Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp935Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp825Bd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp825Ad*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp815Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp805Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v050Dp8053d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3321d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3307d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3305d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3273d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v13D3p3247d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1761p0B05d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1784d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1742d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1732d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0B05p1731d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1690p0740d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v15C5p0008d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C2Cd*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C28d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C27d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C25d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C23d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C12d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C09d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C07d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v14B2p3C06d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p3072d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p3071d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p3070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p2870d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p2770d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v8516p2070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1EDAp2310d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1EDAp2210d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1EDAp2012d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v1482p3C09d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p3072d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p3071d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p3070d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p2770d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v07B8p2870d*dc*dsc*dp*ic*isc*ip*in* rt2800usb
+alias usb:v0CF3p20FFd*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v04DAp3904d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0411p0197d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0411p017Fd*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v083ApA704d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0846p9018d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3p7010d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v1668p1200d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3p7015d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v057Cp8403d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3pB002d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3pB003d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v040Dp3801d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v04CAp4605d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3350d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3349d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3348d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3346d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3328d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v13D3p3327d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v07D1p3A10d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0846p9030d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3p1006d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias usb:v0CF3p9271d*dc*dsc*dp*ic*isc*ip*in* ath9k_htc
+alias platform:wcn36xx wcn36xx
+alias usb:v0586p401Ad*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v1557p8150d*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v07B8p401Ad*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v3980p0003d*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v0411p0012d*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v0BDAp8150d*dc*dsc*dp*ic*isc*ip*in* rtl8150
+alias usb:v04E8pA101d*dc*dsc*dp*ic*isc*ip*in* r8152
+alias usb:v0BDAp8153d*dc*dsc*dp*ic*isc*ip*in* r8152
+alias usb:v0BDAp8152d*dc*dsc*dp*ic*isc*ip*in* r8152
+alias usb:v066Bp20F9d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p172Ad*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p7E2Bd*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0DB0pA877d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v14EApAB11d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p772Ad*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v05ACp1402d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v050Dp5055d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v04BBp0930d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v1737p0039d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v2001p1A02d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v2001p3C05d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v07D1p3C05d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v1557p7720d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v13B1p0018d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0789p0160d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p1780d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p7720d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p772Bd*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v17EFp7203d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v04F1p3008d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v1631p6200d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v1189p0893d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v07AAp0017d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0DF6p0056d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v6189p182Dd*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0411p006Ed*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0411p003Dd*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0557p2009d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v08DDp90FFd*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v07B8p420Ad*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0B95p1720d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v2001p1A00d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v0846p1040d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v077Bp2226d*dc*dsc*dp*ic*isc*ip*in* asix
+alias usb:v17EFp304Bd*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v04E8pA100d*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v0DF6p0072d*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v2001p4A00d*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v0B95p178Ad*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v0B95p1790d*dc*dsc*dp*ic*isc*ip*in* ax88179_178a
+alias usb:v12D1p*d*dc*dsc*dp*ic02isc06ipFFin* cdc_ether
+alias usb:v*p*d*dc*dsc*dp*ic02isc0Aip00in* cdc_ether
+alias usb:v*p*d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v1BC7p*d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v19D2p1181d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v19D2p1177d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v19D2p1173d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v19D2p1015d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v19D2p1003d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04E8pA101d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v0BDAp8153d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v0BDAp8152d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v12D1p14ACd*dc*dsc*dp*ic*isc*ip*in01* cdc_ether
+alias usb:v16D5p650Ad*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v1410p9011d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v413Cp819Bd*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v413Cp8196d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v413Cp8195d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v1410p9010d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v1410pB001d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v046DpC11Fd*dc*dsc*dp*ic02isc0Aip00in* cdc_ether
+alias usb:v1004p61AAd*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v07B4p0F02d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp9050d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp9032d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp9031d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp8007d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp8006d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp8005d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v04DDp8004d*dc*dsc*dp*ic02isc06ip00in* cdc_ether
+alias usb:v06D0p0622d*dc*dsc*dp*ic*isc*ip*in* net1080
+alias usb:v0525p1080d*dc*dsc*dp*ic*isc*ip*in* net1080
+alias usb:v0525pA4A2d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v1286p8001d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v8086p07D3d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v0E7Ep1001d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v049Fp505Ad*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v0525p9901d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v056Cp8100d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v050Dp0004d*dc*dsc*dp*ic*isc*ip*in* cdc_subset
+alias usb:v046DpC11Fd*dc*dsc*dp*ic02isc0Aip00in* zaurus
+alias usb:v07B4p0F02d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v22B8p6425d*dc*dsc*dp*ic02isc0Aip00in* zaurus
+alias usb:v22B8p6027d*dc*dsc*dp*ic02isc0Aip00in* zaurus
+alias usb:v04DDp9050d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp9032d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp9031d*dc*dsc*dp*ic02isc0Aip00in* zaurus
+alias usb:v04DDp9031d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp8007d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp8006d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp8005d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v04DDp8004d*dc*dsc*dp*ic02isc06ip00in* zaurus
+alias usb:v*p*d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
+alias usb:v1519p0443d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
+alias usb:v12D1p*d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
+alias usb:v0930p*d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
+alias usb:v413Cp*d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
+alias usb:v0BDBp*d*dc*dsc*dp*ic02isc0Dip00in* cdc_ncm
--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.builtin
@@ -0,0 +1,483 @@
+kernel/arch/arm64/crypto/sha2-ce.ko
+kernel/arch/arm64/crypto/aes-ce-cipher.ko
+kernel/arch/arm64/crypto/aes-ce-blk.ko
+kernel/kernel/configs.ko
+kernel/mm/zsmalloc.ko
+kernel/fs/binfmt_script.ko
+kernel/fs/mbcache.ko
+kernel/fs/autofs4/autofs4.ko
+kernel/fs/configfs/configfs.ko
+kernel/fs/ecryptfs/ecryptfs.ko
+kernel/fs/exportfs/exportfs.ko
+kernel/fs/ext4/ext4.ko
+kernel/fs/fat/fat.ko
+kernel/fs/fat/vfat.ko
+kernel/fs/fat/msdos.ko
+kernel/fs/fuse/fuse.ko
+kernel/fs/isofs/isofs.ko
+kernel/fs/jbd2/jbd2.ko
+kernel/fs/nls/nls_base.ko
+kernel/fs/nls/nls_cp437.ko
+kernel/fs/nls/nls_cp950.ko
+kernel/fs/nls/nls_ascii.ko
+kernel/fs/nls/nls_iso8859-1.ko
+kernel/fs/nls/nls_utf8.ko
+kernel/fs/pstore/ramoops.ko
+kernel/fs/sdcardfs/sdcardfs.ko
+kernel/security/keys/encrypted-keys/encrypted-keys.ko
+kernel/crypto/crypto.ko
+kernel/crypto/crypto_wq.ko
+kernel/crypto/crypto_algapi.ko
+kernel/crypto/aead.ko
+kernel/crypto/crypto_blkcipher.ko
+kernel/crypto/chainiv.ko
+kernel/crypto/eseqiv.ko
+kernel/crypto/seqiv.ko
+kernel/crypto/crypto_hash.ko
+kernel/crypto/pcompress.ko
+kernel/crypto/cryptomgr.ko
+kernel/crypto/hmac.ko
+kernel/crypto/xcbc.ko
+kernel/crypto/crypto_null.ko
+kernel/crypto/md5.ko
+kernel/crypto/sha1_generic.ko
+kernel/crypto/sha256_generic.ko
+kernel/crypto/sha512_generic.ko
+kernel/crypto/ecb.ko
+kernel/crypto/cbc.ko
+kernel/crypto/ctr.ko
+kernel/crypto/ccm.ko
+kernel/crypto/cryptd.ko
+kernel/crypto/des_generic.ko
+kernel/crypto/twofish_generic.ko
+kernel/crypto/twofish_common.ko
+kernel/crypto/aes_generic.ko
+kernel/crypto/arc4.ko
+kernel/crypto/deflate.ko
+kernel/crypto/crc32c_generic.ko
+kernel/crypto/authenc.ko
+kernel/crypto/authencesn.ko
+kernel/crypto/lzo.ko
+kernel/crypto/rng.ko
+kernel/crypto/krng.ko
+kernel/crypto/ablk_helper.ko
+kernel/block/deadline-iosched.ko
+kernel/block/cfq-iosched.ko
+kernel/drivers/base/firmware_class.ko
+kernel/drivers/base/regmap/regmap-i2c.ko
+kernel/drivers/base/regmap/regmap-spi.ko
+kernel/drivers/base/regmap/regmap-mmio.ko
+kernel/drivers/block/loop.ko
+kernel/drivers/block/zram/zram.ko
+kernel/drivers/bluetooth/hci_uart.ko
+kernel/drivers/cpufreq/cpufreq_stats.ko
+kernel/drivers/cpufreq/cpufreq_performance.ko
+kernel/drivers/cpufreq/cpufreq_powersave.ko
+kernel/drivers/cpufreq/cpufreq_userspace.ko
+kernel/drivers/cpufreq/cpufreq_ondemand.ko
+kernel/drivers/cpufreq/cpufreq_interactive.ko
+kernel/drivers/hid/hid.ko
+kernel/drivers/hid/uhid.ko
+kernel/drivers/hid/hid-generic.ko
+kernel/drivers/hid/hid-a4tech.ko
+kernel/drivers/hid/hid-apple.ko
+kernel/drivers/hid/hid-belkin.ko
+kernel/drivers/hid/hid-cherry.ko
+kernel/drivers/hid/hid-chicony.ko
+kernel/drivers/hid/hid-cypress.ko
+kernel/drivers/hid/hid-dr.ko
+kernel/drivers/hid/hid-ezkey.ko
+kernel/drivers/hid/hid-gyration.ko
+kernel/drivers/hid/hid-kensington.ko
+kernel/drivers/hid/hid-kye.ko
+kernel/drivers/hid/hid-logitech.ko
+kernel/drivers/hid/hid-magicmouse.ko
+kernel/drivers/hid/hid-microsoft.ko
+kernel/drivers/hid/hid-monterey.ko
+kernel/drivers/hid/hid-pl.ko
+kernel/drivers/hid/hid-petalynx.ko
+kernel/drivers/hid/hid-samsung.ko
+kernel/drivers/hid/hid-sjoy.ko
+kernel/drivers/hid/hid-sunplus.ko
+kernel/drivers/hid/hid-gaff.ko
+kernel/drivers/hid/hid-tmff.ko
+kernel/drivers/hid/hid-topseed.ko
+kernel/drivers/hid/hid-twinhan.ko
+kernel/drivers/hid/hid-zpff.ko
+kernel/drivers/hid/usbhid/usbhid.ko
+kernel/drivers/i2c/i2c-core.ko
+kernel/drivers/i2c/busses/i2c-mtk.ko
+kernel/drivers/i2c/busses/i2c_mtk_debug.ko
+kernel/drivers/input/input-core.ko
+kernel/drivers/input/evdev.ko
+kernel/drivers/input/keyboard/mediatek/kpd.ko
+kernel/drivers/input/keyboard/mediatek/mt6797/hal_kpd.ko
+kernel/drivers/input/misc/uinput.ko
+kernel/drivers/input/touchscreen/of_touchscreen.ko
+kernel/drivers/input/touchscreen/mediatek/tpd_common_probe.ko
+kernel/drivers/leds/led-class.ko
+kernel/drivers/leds/trigger/ledtrig-timer.ko
+kernel/drivers/md/dm-mod.ko
+kernel/drivers/md/dm-bufio.ko
+kernel/drivers/md/dm-crypt.ko
+kernel/drivers/md/dm-verity.ko
+kernel/drivers/misc/uid_cputime.ko
+kernel/drivers/misc/eeprom/eeprom_93cx6.ko
+kernel/drivers/misc/mediatek/accdet/accdet_drv.ko
+kernel/drivers/misc/mediatek/accdet/mt6797/accdet.ko
+kernel/drivers/misc/mediatek/aee/aed/aed.ko
+kernel/drivers/misc/mediatek/aee/ipanic/ipanic.ko
+kernel/drivers/misc/mediatek/aee/mrdump/mrdump_mini.ko
+kernel/drivers/misc/mediatek/aee/mrdump/mrdump_control.ko
+kernel/drivers/misc/mediatek/auxadc/mt_auxadc.ko
+kernel/drivers/misc/mediatek/connectivity/ant/stp_chrdev_ant.ko
+kernel/drivers/misc/mediatek/connectivity/drv_bt/hci_stp.ko
+kernel/drivers/misc/mediatek/connectivity/gps/stp_chrdev_gps.ko
+kernel/drivers/misc/mediatek/connectivity/gps/gps.ko
+kernel/drivers/misc/mediatek/connectivity/wlan/wmt_chrdev_wifi.ko
+kernel/drivers/misc/mediatek/connectivity/wlan/gen3/wlan_gen3.ko
+kernel/drivers/misc/mediatek/dramc/mt6797/mt_dramc.ko
+kernel/drivers/misc/mediatek/gpio/mt_gpio_core.ko
+kernel/drivers/misc/mediatek/gpio/aeon_gpio.ko
+kernel/drivers/misc/mediatek/leds/leds_drv.ko
+kernel/drivers/misc/mediatek/leds/mt6797/leds.ko
+kernel/drivers/misc/mediatek/m4u/mt6797/m4u_hw.ko
+kernel/drivers/misc/mediatek/power/mt6797/battery_meter_hal.ko
+kernel/drivers/misc/mediatek/power/mt6797/mtk_bif_intf.ko
+kernel/drivers/misc/mediatek/power/mt6797/rt9466.ko
+kernel/drivers/misc/mediatek/power/mt6797/mtk_charger_intf.ko
+kernel/drivers/misc/mediatek/power/mt6797/bq25890.ko
+kernel/drivers/misc/mediatek/power/mt6797/charging_hw_bq25890.ko
+kernel/drivers/misc/mediatek/pwm/mt_pwm.ko
+kernel/drivers/misc/mediatek/pwm/mt6797/mt_pwm_hal.ko
+kernel/drivers/misc/mediatek/thermal/mtk_thermal_monitor.ko
+kernel/drivers/misc/mediatek/thermal/mtk_cooler_shutdown.ko
+kernel/drivers/misc/mediatek/thermal/mtk_cooler_kshutdown.ko
+kernel/drivers/misc/mediatek/thermal/mtk_cooler_cam.ko
+kernel/drivers/misc/mediatek/thermal/mtk_cooler_vrt.ko
+kernel/drivers/misc/mediatek/thermal/common/mtk_thermal_platform.ko
+kernel/drivers/misc/mediatek/thermal/common/ap_thermal_limit.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_amutt.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_mutt.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_fps.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_bcct.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_atm.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_dtm.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_cooler_sysrst.ko
+kernel/drivers/misc/mediatek/thermal/common/coolers/mtk_ta.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_battery.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_bts.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_btsmdpa.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_cpu.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_pa.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_pa_thput.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_pmic.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/ts_da9214.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_dram.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_imgsensor.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts_wmt.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts1.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts2.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts3.ko
+kernel/drivers/misc/mediatek/thermal/common/thermal_zones/mtk_ts4.ko
+kernel/drivers/misc/mediatek/thermal/mt6797/src/mtk_tc.ko
+kernel/drivers/misc/mediatek/thermal/mt6797/src/mtk_pmic_efuse.ko
+kernel/drivers/misc/mediatek/uart/uart.ko
+kernel/drivers/misc/mediatek/uart/mt6797/platform_uart.ko
+kernel/drivers/misc/mediatek/vibrator/vibrator_drv.ko
+kernel/drivers/misc/mediatek/vibrator/mt6797/vibrator.ko
+kernel/drivers/misc/mediatek/xhci/xhci-mtk-power.ko
+kernel/drivers/misc/mediatek/xhci/xhci-mtk-driver.ko
+kernel/drivers/misc/mediatek/xhci/xhci-mtk-attrs.ko
+kernel/drivers/mmc/card/mmc_block.ko
+kernel/drivers/mmc/core/mmc_core.ko
+kernel/drivers/mmc/host/mediatek/emmc_rpmb.ko
+kernel/drivers/mmc/host/mediatek/mt6797/sd.ko
+kernel/drivers/mmc/host/mediatek/mt6797/dbg.ko
+kernel/drivers/mmc/host/mediatek/mt6797/mt_sd_misc.ko
+kernel/drivers/mmc/host/mediatek/mt6797/msdc_tune.ko
+kernel/drivers/mmc/host/mediatek/mt6797/msdc_partition.ko
+kernel/drivers/mmc/host/mediatek/mt6797/msdc_io.ko
+kernel/drivers/mmc/host/mediatek/mt6797/autok.ko
+kernel/drivers/mtd/mtd.ko
+kernel/drivers/mtd/ofpart.ko
+kernel/drivers/mtd/mtd_blkdevs.ko
+kernel/drivers/mtd/mtdblock.ko
+kernel/drivers/mtd/chips/chipreg.ko
+kernel/drivers/mtd/nand/nand_ids.ko
+kernel/drivers/net/ifb.ko
+kernel/drivers/net/macvlan.ko
+kernel/drivers/net/tun.ko
+kernel/drivers/net/veth.ko
+kernel/drivers/net/ppp/ppp_generic.ko
+kernel/drivers/net/ppp/ppp_async.ko
+kernel/drivers/net/ppp/bsd_comp.ko
+kernel/drivers/net/ppp/ppp_deflate.ko
+kernel/drivers/net/ppp/ppp_mppe.ko
+kernel/drivers/net/ppp/ppp_synctty.ko
+kernel/drivers/net/ppp/pppox.ko
+kernel/drivers/net/ppp/pppoe.ko
+kernel/drivers/net/ppp/pppolac.ko
+kernel/drivers/net/ppp/pppopns.ko
+kernel/drivers/net/slip/slhc.ko
+kernel/drivers/power/mediatek/battery_meter_fg_20.ko
+kernel/drivers/power/mediatek/battery_common_fg_20.ko
+kernel/drivers/power/mediatek/switch_charging.ko
+kernel/drivers/scsi/scsi_mod.ko
+kernel/drivers/scsi/sd_mod.ko
+kernel/drivers/spi/mediatek/mt6797/spi.ko
+kernel/drivers/spi/mediatek/mt6797/spi-dev.ko
+kernel/drivers/staging/android/ion/mtk/ion_drv.ko
+kernel/drivers/staging/android/ion/mtk/ion_mm_heap.ko
+kernel/drivers/staging/android/ion/mtk/ion_fb_heap.ko
+kernel/drivers/staging/android/ion/mtk/ion_sec_heap.ko
+kernel/drivers/switch/switch_class.ko
+kernel/drivers/thermal/thermal_sys.ko
+kernel/drivers/tty/serial/serial_core.ko
+kernel/drivers/usb/common/usb-common.ko
+kernel/drivers/usb/core/usbcore.ko
+kernel/drivers/usb/gadget/libcomposite.ko
+kernel/drivers/usb/gadget/function/usb_f_acm.ko
+kernel/drivers/usb/gadget/function/u_serial.ko
+kernel/drivers/usb/gadget/function/usb_f_serial.ko
+kernel/drivers/usb/gadget/function/usb_f_mass_storage.ko
+kernel/drivers/usb/gadget/function/usb_f_fs.ko
+kernel/drivers/usb/gadget/function/usb_f_audio_source.ko
+kernel/drivers/usb/gadget/udc/udc-core.ko
+kernel/drivers/usb/host/xhci-plat-hcd.ko
+kernel/drivers/usb/host/xhci-hcd.ko
+kernel/drivers/usb/misc/trancevibrator.ko
+kernel/drivers/usb/phy/phy-generic.ko
+kernel/drivers/usb/storage/usb-storage.ko
+kernel/drivers/usb/storage/ums-alauda.ko
+kernel/drivers/usb/storage/ums-cypress.ko
+kernel/drivers/usb/storage/ums-datafab.ko
+kernel/drivers/usb/storage/ums-freecom.ko
+kernel/drivers/usb/storage/ums-isd200.ko
+kernel/drivers/usb/storage/ums-jumpshot.ko
+kernel/drivers/usb/storage/ums-karma.ko
+kernel/drivers/usb/storage/ums-onetouch.ko
+kernel/drivers/usb/storage/ums-sddr09.ko
+kernel/drivers/usb/storage/ums-sddr55.ko
+kernel/drivers/usb/storage/ums-usbat.ko
+kernel/drivers/video/fbdev/core/fb.ko
+kernel/drivers/video/fbdev/core/cfbfillrect.ko
+kernel/drivers/video/fbdev/core/cfbcopyarea.ko
+kernel/drivers/video/fbdev/core/cfbimgblt.ko
+kernel/drivers/watchdog/mediatek/wdk/wd_common_drv.ko
+kernel/sound/soundcore.ko
+kernel/sound/sound_firmware.ko
+kernel/sound/core/snd.ko
+kernel/sound/core/snd-hwdep.ko
+kernel/sound/core/snd-timer.ko
+kernel/sound/core/snd-hrtimer.ko
+kernel/sound/core/snd-pcm.ko
+kernel/sound/core/snd-rawmidi.ko
+kernel/sound/core/snd-compress.ko
+kernel/sound/core/oss/snd-mixer-oss.ko
+kernel/sound/core/oss/snd-pcm-oss.ko
+kernel/sound/core/seq/snd-seq.ko
+kernel/sound/core/seq/snd-seq-device.ko
+kernel/sound/core/seq/snd-seq-midi-event.ko
+kernel/sound/core/seq/snd-seq-dummy.ko
+kernel/sound/core/seq/snd-seq-midi.ko
+kernel/sound/core/seq/oss/snd-seq-oss.ko
+kernel/sound/soc/snd-soc-core.ko
+kernel/sound/soc/generic/snd-soc-simple-card.ko
+kernel/sound/usb/snd-usb-audio.ko
+kernel/sound/usb/snd-usbmidi-lib.ko
+kernel/net/802/p8022.ko
+kernel/net/802/psnap.ko
+kernel/net/802/stp.ko
+kernel/net/8021q/vlan_core.ko
+kernel/net/8021q/8021q.ko
+kernel/net/bluetooth/bluetooth.ko
+kernel/net/bluetooth/bnep/bnep.ko
+kernel/net/bluetooth/hidp/hidp.ko
+kernel/net/bluetooth/rfcomm/rfcomm.ko
+kernel/net/bridge/bridge.ko
+kernel/net/bridge/br_netfilter.ko
+kernel/net/dns_resolver/dns_resolver.ko
+kernel/net/ipv4/ip_tunnel.ko
+kernel/net/ipv4/ipip.ko
+kernel/net/ipv4/ah4.ko
+kernel/net/ipv4/esp4.ko
+kernel/net/ipv4/ipcomp.ko
+kernel/net/ipv4/xfrm4_tunnel.ko
+kernel/net/ipv4/xfrm4_mode_beet.ko
+kernel/net/ipv4/tunnel4.ko
+kernel/net/ipv4/xfrm4_mode_transport.ko
+kernel/net/ipv4/xfrm4_mode_tunnel.ko
+kernel/net/ipv4/inet_diag.ko
+kernel/net/ipv4/tcp_diag.ko
+kernel/net/ipv4/udp_diag.ko
+kernel/net/ipv4/tcp_cubic.ko
+kernel/net/ipv4/netfilter/nf_conntrack_ipv4.ko
+kernel/net/ipv4/netfilter/nf_nat_ipv4.ko
+kernel/net/ipv4/netfilter/nf_defrag_ipv4.ko
+kernel/net/ipv4/netfilter/nf_log_ipv4.ko
+kernel/net/ipv4/netfilter/nf_reject_ipv4.ko
+kernel/net/ipv4/netfilter/nf_nat_pptp.ko
+kernel/net/ipv4/netfilter/nf_nat_masquerade_ipv4.ko
+kernel/net/ipv4/netfilter/nf_nat_proto_gre.ko
+kernel/net/ipv4/netfilter/ip_tables.ko
+kernel/net/ipv4/netfilter/iptable_filter.ko
+kernel/net/ipv4/netfilter/iptable_mangle.ko
+kernel/net/ipv4/netfilter/iptable_nat.ko
+kernel/net/ipv4/netfilter/iptable_raw.ko
+kernel/net/ipv4/netfilter/ipt_MASQUERADE.ko
+kernel/net/ipv4/netfilter/ipt_REJECT.ko
+kernel/net/ipv4/netfilter/arp_tables.ko
+kernel/net/ipv6/ipv6.ko
+kernel/net/ipv6/ah6.ko
+kernel/net/ipv6/esp6.ko
+kernel/net/ipv6/ipcomp6.ko
+kernel/net/ipv6/xfrm6_tunnel.ko
+kernel/net/ipv6/tunnel6.ko
+kernel/net/ipv6/xfrm6_mode_transport.ko
+kernel/net/ipv6/xfrm6_mode_tunnel.ko
+kernel/net/ipv6/xfrm6_mode_ro.ko
+kernel/net/ipv6/xfrm6_mode_beet.ko
+kernel/net/ipv6/mip6.ko
+kernel/net/ipv6/sit.ko
+kernel/net/ipv6/ip6_tunnel.ko
+kernel/net/ipv6/inet6_hashtables.ko
+kernel/net/ipv6/netfilter/ip6_tables.ko
+kernel/net/ipv6/netfilter/ip6table_filter.ko
+kernel/net/ipv6/netfilter/ip6table_mangle.ko
+kernel/net/ipv6/netfilter/ip6table_raw.ko
+kernel/net/ipv6/netfilter/ip6table_security.ko
+kernel/net/ipv6/netfilter/ip6table_nat.ko
+kernel/net/ipv6/netfilter/nf_conntrack_ipv6.ko
+kernel/net/ipv6/netfilter/nf_nat_ipv6.ko
+kernel/net/ipv6/netfilter/nf_nat_masquerade_ipv6.ko
+kernel/net/ipv6/netfilter/nf_defrag_ipv6.ko
+kernel/net/ipv6/netfilter/nf_log_ipv6.ko
+kernel/net/ipv6/netfilter/nf_reject_ipv6.ko
+kernel/net/ipv6/netfilter/ip6t_ah.ko
+kernel/net/ipv6/netfilter/ip6t_eui64.ko
+kernel/net/ipv6/netfilter/ip6t_frag.ko
+kernel/net/ipv6/netfilter/ip6t_ipv6header.ko
+kernel/net/ipv6/netfilter/ip6t_mh.ko
+kernel/net/ipv6/netfilter/ip6t_hbh.ko
+kernel/net/ipv6/netfilter/ip6t_rpfilter.ko
+kernel/net/ipv6/netfilter/ip6t_rt.ko
+kernel/net/ipv6/netfilter/ip6t_MASQUERADE.ko
+kernel/net/ipv6/netfilter/ip6t_REJECT.ko
+kernel/net/key/af_key.ko
+kernel/net/llc/llc.ko
+kernel/net/mac80211/mac80211.ko
+kernel/net/netfilter/nfnetlink.ko
+kernel/net/netfilter/nfnetlink_acct.ko
+kernel/net/netfilter/nfnetlink_queue.ko
+kernel/net/netfilter/nfnetlink_log.ko
+kernel/net/netfilter/nf_conntrack.ko
+kernel/net/netfilter/nf_conntrack_proto_gre.ko
+kernel/net/netfilter/nf_conntrack_proto_udplite.ko
+kernel/net/netfilter/nf_conntrack_netlink.ko
+kernel/net/netfilter/nf_conntrack_ftp.ko
+kernel/net/netfilter/nf_conntrack_pptp.ko
+kernel/net/netfilter/nf_conntrack_tftp.ko
+kernel/net/netfilter/nf_log_common.ko
+kernel/net/netfilter/nf_nat.ko
+kernel/net/netfilter/nf_nat_proto_udplite.ko
+kernel/net/netfilter/nf_nat_ftp.ko
+kernel/net/netfilter/nf_nat_tftp.ko
+kernel/net/netfilter/x_tables.ko
+kernel/net/netfilter/xt_tcpudp.ko
+kernel/net/netfilter/xt_mark.ko
+kernel/net/netfilter/xt_connmark.ko
+kernel/net/netfilter/xt_nat.ko
+kernel/net/netfilter/xt_AUDIT.ko
+kernel/net/netfilter/xt_CHECKSUM.ko
+kernel/net/netfilter/xt_CLASSIFY.ko
+kernel/net/netfilter/xt_CT.ko
+kernel/net/netfilter/xt_DSCP.ko
+kernel/net/netfilter/xt_HL.ko
+kernel/net/netfilter/xt_LED.ko
+kernel/net/netfilter/xt_LOG.ko
+kernel/net/netfilter/xt_NETMAP.ko
+kernel/net/netfilter/xt_NFLOG.ko
+kernel/net/netfilter/xt_NFQUEUE.ko
+kernel/net/netfilter/xt_RATEEST.ko
+kernel/net/netfilter/xt_REDIRECT.ko
+kernel/net/netfilter/xt_SECMARK.ko
+kernel/net/netfilter/xt_TPROXY.ko
+kernel/net/netfilter/xt_TCPMSS.ko
+kernel/net/netfilter/xt_TCPOPTSTRIP.ko
+kernel/net/netfilter/xt_TEE.ko
+kernel/net/netfilter/xt_TRACE.ko
+kernel/net/netfilter/xt_IDLETIMER.ko
+kernel/net/netfilter/xt_addrtype.ko
+kernel/net/netfilter/xt_cluster.ko
+kernel/net/netfilter/xt_comment.ko
+kernel/net/netfilter/xt_connbytes.ko
+kernel/net/netfilter/xt_connlimit.ko
+kernel/net/netfilter/xt_conntrack.ko
+kernel/net/netfilter/xt_cpu.ko
+kernel/net/netfilter/xt_dccp.ko
+kernel/net/netfilter/xt_devgroup.ko
+kernel/net/netfilter/xt_dscp.ko
+kernel/net/netfilter/xt_ecn.ko
+kernel/net/netfilter/xt_esp.ko
+kernel/net/netfilter/xt_hashlimit.ko
+kernel/net/netfilter/xt_helper.ko
+kernel/net/netfilter/xt_hl.ko
+kernel/net/netfilter/xt_iprange.ko
+kernel/net/netfilter/xt_length.ko
+kernel/net/netfilter/xt_limit.ko
+kernel/net/netfilter/xt_mac.ko
+kernel/net/netfilter/xt_multiport.ko
+kernel/net/netfilter/xt_nfacct.ko
+kernel/net/netfilter/xt_osf.ko
+kernel/net/netfilter/xt_owner.ko
+kernel/net/netfilter/xt_physdev.ko
+kernel/net/netfilter/xt_pkttype.ko
+kernel/net/netfilter/xt_policy.ko
+kernel/net/netfilter/xt_quota.ko
+kernel/net/netfilter/xt_quota2.ko
+kernel/net/netfilter/xt_rateest.ko
+kernel/net/netfilter/xt_realm.ko
+kernel/net/netfilter/xt_recent.ko
+kernel/net/netfilter/xt_sctp.ko
+kernel/net/netfilter/xt_socket.ko
+kernel/net/netfilter/xt_state.ko
+kernel/net/netfilter/xt_statistic.ko
+kernel/net/netfilter/xt_string.ko
+kernel/net/netfilter/xt_tcpmss.ko
+kernel/net/netfilter/xt_time.ko
+kernel/net/netfilter/xt_u32.ko
+kernel/net/netlink/netlink_diag.ko
+kernel/net/packet/af_packet.ko
+kernel/net/packet/af_packet_diag.ko
+kernel/net/sched/act_police.ko
+kernel/net/sched/act_mirred.ko
+kernel/net/sched/act_ipt.ko
+kernel/net/sched/sch_htb.ko
+kernel/net/sched/sch_ingress.ko
+kernel/net/sched/cls_u32.ko
+kernel/net/sched/cls_fw.ko
+kernel/net/sched/cls_flow.ko
+kernel/net/unix/unix.ko
+kernel/net/unix/unix_diag.ko
+kernel/net/wireless/cfg80211.ko
+kernel/net/xfrm/xfrm_algo.ko
+kernel/net/xfrm/xfrm_user.ko
+kernel/net/xfrm/xfrm_ipcomp.ko
+kernel/lib/bitrev.ko
+kernel/lib/crc-ccitt.ko
+kernel/lib/crc16.ko
+kernel/lib/crc-itu-t.ko
+kernel/lib/crc32.ko
+kernel/lib/libcrc32c.ko
+kernel/lib/ts_kmp.ko
+kernel/lib/ts_bm.ko
+kernel/lib/ts_fsm.ko
+kernel/lib/digsig.ko
+kernel/lib/fonts/font.ko
+kernel/lib/lzo/lzo_compress.ko
+kernel/lib/lzo/lzo_decompress.ko
+kernel/lib/mpi/mpi.ko
+kernel/lib/reed_solomon/reed_solomon.ko
+kernel/lib/zlib_deflate/zlib_deflate.ko
+kernel/lib/zlib_inflate/zlib_inflate.ko
--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.dep
@@ -0,0 +1,32 @@
+kernel/drivers/net/wireless/rtl8812au/8812au.ko:
+kernel/drivers/net/wireless/rtl818x/rtl8187/rtl8187.ko:
+kernel/drivers/net/wireless/rtlwifi/rtlwifi.ko:
+kernel/drivers/net/wireless/rtlwifi/rtl_usb.ko: kernel/drivers/net/wireless/rtlwifi/rtlwifi.ko
+kernel/drivers/net/wireless/rtlwifi/rtl8192c/rtl8192c-common.ko: kernel/drivers/net/wireless/rtlwifi/rtlwifi.ko
+kernel/drivers/net/wireless/rtlwifi/rtl8192cu/rtl8192cu.ko: kernel/drivers/net/wireless/rtlwifi/rtl_usb.ko kernel/drivers/net/wireless/rtlwifi/rtl8192c/rtl8192c-common.ko kernel/drivers/net/wireless/rtlwifi/rtlwifi.ko
+kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko:
+kernel/drivers/net/wireless/rt2x00/rt2x00usb.ko: kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/rt2x00/rt2800lib.ko: kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/rt2x00/rt2500usb.ko: kernel/drivers/net/wireless/rt2x00/rt2x00usb.ko kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/rt2x00/rt73usb.ko: kernel/drivers/net/wireless/rt2x00/rt2x00usb.ko kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/rt2x00/rt2800usb.ko: kernel/drivers/net/wireless/rt2x00/rt2800lib.ko kernel/drivers/net/wireless/rt2x00/rt2x00usb.ko kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k.ko: kernel/drivers/net/wireless/ath/ath9k/ath9k_common.ko kernel/drivers/net/wireless/ath/ath9k/ath9k_hw.ko kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_hw.ko: kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_common.ko: kernel/drivers/net/wireless/ath/ath9k/ath9k_hw.ko kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_htc.ko: kernel/drivers/net/wireless/ath/ath9k/ath9k_common.ko kernel/drivers/net/wireless/ath/ath9k/ath9k_hw.ko kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/wireless/ath/ath10k/ath10k_core.ko: kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/wireless/ath/wcn36xx/wcn36xx.ko:
+kernel/drivers/net/wireless/ath/ath.ko:
+kernel/drivers/net/usb/rtl8150.ko:
+kernel/drivers/net/usb/r8152.ko: kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/asix.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/phy/libphy.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/ax88179_178a.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/cdc_ether.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/net1080.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/cdc_subset.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/zaurus.ko: kernel/drivers/net/usb/cdc_ether.ko kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/usbnet.ko: kernel/drivers/net/mii.ko
+kernel/drivers/net/usb/cdc_ncm.ko: kernel/drivers/net/usb/usbnet.ko kernel/drivers/net/mii.ko
+kernel/drivers/net/mii.ko:
+kernel/drivers/net/phy/libphy.ko:
+kernel/drivers/of/of_mdio.ko: kernel/drivers/net/phy/libphy.ko
--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.order
@@ -0,0 +1,32 @@
+kernel/drivers/net/wireless/rtl8812au/8812au.ko
+kernel/drivers/net/wireless/rtl818x/rtl8187/rtl8187.ko
+kernel/drivers/net/wireless/rtlwifi/rtlwifi.ko
+kernel/drivers/net/wireless/rtlwifi/rtl_usb.ko
+kernel/drivers/net/wireless/rtlwifi/rtl8192c/rtl8192c-common.ko
+kernel/drivers/net/wireless/rtlwifi/rtl8192cu/rtl8192cu.ko
+kernel/drivers/net/wireless/rt2x00/rt2x00lib.ko
+kernel/drivers/net/wireless/rt2x00/rt2x00usb.ko
+kernel/drivers/net/wireless/rt2x00/rt2800lib.ko
+kernel/drivers/net/wireless/rt2x00/rt2500usb.ko
+kernel/drivers/net/wireless/rt2x00/rt73usb.ko
+kernel/drivers/net/wireless/rt2x00/rt2800usb.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_hw.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_common.ko
+kernel/drivers/net/wireless/ath/ath9k/ath9k_htc.ko
+kernel/drivers/net/wireless/ath/ath10k/ath10k_core.ko
+kernel/drivers/net/wireless/ath/wcn36xx/wcn36xx.ko
+kernel/drivers/net/wireless/ath/ath.ko
+kernel/drivers/net/usb/rtl8150.ko
+kernel/drivers/net/usb/r8152.ko
+kernel/drivers/net/usb/asix.ko
+kernel/drivers/net/usb/ax88179_178a.ko
+kernel/drivers/net/usb/cdc_ether.ko
+kernel/drivers/net/usb/net1080.ko
+kernel/drivers/net/usb/cdc_subset.ko
+kernel/drivers/net/usb/zaurus.ko
+kernel/drivers/net/usb/usbnet.ko
+kernel/drivers/net/usb/cdc_ncm.ko
+kernel/drivers/net/mii.ko
+kernel/drivers/net/phy/libphy.ko
+kernel/drivers/of/of_mdio.ko
--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.softdep
@@ -0,0 +1 @@
+# Soft dependencies extracted from modules themselves.
--- /dev/null
+++ kali-gemini-linux-3.18.41.9/lib/modules/3.18.41-kali+/modules.symbols
@@ -0,0 +1,469 @@
+# Aliases for symbols, used by symbol_request().
+alias symbol:ar9003_paprd_is_done ath9k_hw
+alias symbol:_rtl92c_phy_bb8192c_config_parafile rtl8192c_common
+alias symbol:rt2x00mac_get_stats rt2x00lib
+alias symbol:ath9k_hw_gettsf32 ath9k_hw
+alias symbol:usbnet_get_msglevel usbnet
+alias symbol:rt2x00mac_set_tim rt2x00lib
+alias symbol:rtl_ps_set_rf_state rtlwifi
+alias symbol:ath9k_cmn_debug_phy_err ath9k_common
+alias symbol:rtl_phy_scan_operation_backup rtlwifi
+alias symbol:ath_is_49ghz_allowed ath
+alias symbol:rt2800_write_tx_data rt2800lib
+alias symbol:cdc_ncm_fill_tx_frame cdc_ncm
+alias symbol:genphy_read_status libphy
+alias symbol:ath10k_warn ath10k_core
+alias symbol:phy_get_eee_err libphy
+alias symbol:rtl_efuse_shadow_map_update rtlwifi
+alias symbol:rtl_dbgp_flag_init rtlwifi
+alias symbol:rtl_init_rfkill rtlwifi
+alias symbol:ath_hw_setbssidmask ath
+alias symbol:phy_ethtool_gset libphy
+alias symbol:rt2x00lib_probe_dev rt2x00lib
+alias symbol:ath9k_hw_reset_tsf ath9k_hw
+alias symbol:ath9k_cmn_count_streams ath9k_common
+alias symbol:rtl_cam_delete_one_entry rtlwifi
+alias symbol:ath9k_hw_get_tsf_offset ath9k_hw
+alias symbol:rtl92c_dm_watchdog rtl8192c_common
+alias symbol:genphy_config_aneg libphy
+alias symbol:rt2800_get_survey rt2800lib
+alias symbol:rt2x00queue_for_each_entry rt2x00lib
+alias symbol:ath_reg_notifier_apply ath
+alias symbol:rt2x00lib_dmadone rt2x00lib
+alias symbol:_rtl92c_phy_set_rf_sleep rtl8192c_common
+alias symbol:ar9003_paprd_init_table ath9k_hw
+alias symbol:ar9003_hw_bb_watchdog_dbg_info ath9k_hw
+alias symbol:usbnet_pause_rx usbnet
+alias symbol:rt2800_write_beacon rt2800lib
+alias symbol:rt2800_sta_add rt2800lib
+alias symbol:ath9k_hw_ani_monitor ath9k_hw
+alias symbol:rtl_beacon_statistic rtlwifi
+alias symbol:ath9k_hw_beaconq_setup ath9k_hw
+alias symbol:rtl92c_phy_set_bw_mode rtl8192c_common
+alias symbol:ath9k_hw_setpower ath9k_hw
+alias symbol:ar9003_hw_bb_watchdog_check ath9k_hw
+alias symbol:usbnet_write_cmd usbnet
+alias symbol:ath9k_hw_puttxbuf ath9k_hw
+alias symbol:genphy_soft_reset libphy
+alias symbol:rt2x00usb_probe rt2x00usb
+alias symbol:devm_mdiobus_free libphy
+alias symbol:rt2800_config_pairwise_key rt2800lib
+alias symbol:ath9k_hw_updatetxtriglevel ath9k_hw
+alias symbol:phy_ethtool_set_wol libphy
+alias symbol:phy_mii_ioctl libphy
+alias symbol:efuse_shadow_read rtlwifi
+alias symbol:ath9k_hw_check_alive ath9k_hw
+alias symbol:ar9003_hw_disable_phy_restart ath9k_hw
+alias symbol:rt2x00mac_rfkill_poll rt2x00lib
+alias symbol:rt2x00mac_get_ringparam rt2x00lib
+alias symbol:rt2800_wait_csr_ready rt2800lib
+alias symbol:rtl92c_phy_iq_calibrate rtl8192c_common
+alias symbol:dm_restorepowerindex rtl8192c_common
+alias symbol:dfs_pattern_detector_init ath
+alias symbol:usbnet_set_msglevel usbnet
+alias symbol:rtl_deinit_deferred_work rtlwifi
+alias symbol:rt2800_mcu_request rt2800lib
+alias symbol:ar9003_is_paprd_enabled ath9k_hw
+alias symbol:of_mdio_find_bus libphy
+alias symbol:ath9k_hw_set_interrupts ath9k_hw
+alias symbol:rtl_recognize_peer rtlwifi
+alias symbol:ath9k_cmn_debug_base_eeprom ath9k_common
+alias symbol:rt2800_efuse_detect rt2800lib
+alias symbol:ath9k_cmn_debug_recv ath9k_common
+alias symbol:rtl_global_var rtlwifi
+alias symbol:ath9k_cmn_get_channel ath9k_common
+alias symbol:usbnet_get_ethernet_addr usbnet
+alias symbol:rt2800_wait_wpdma_ready rt2800lib
+alias symbol:mdio_bus_type libphy
+alias symbol:usbnet_resume usbnet
+alias symbol:ar9003_mci_set_bt_version ath9k_hw
+alias symbol:ath9k_hw_set_sta_beacon_timers ath9k_hw
+alias symbol:rtl_init_core rtlwifi
+alias symbol:mdiobus_write libphy
+alias symbol:ar9003_paprd_populate_single_table ath9k_hw
+alias symbol:rt2x00mac_get_antenna rt2x00lib
+alias symbol:rtl92c_phy_ap_calibrate rtl8192c_common
+alias symbol:rt2x00mac_conf_tx rt2x00lib
+alias symbol:rt2x00usb_suspend rt2x00usb
+alias symbol:rt2800_config_filter rt2800lib
+alias symbol:ath9k_hw_set_tsfadjust ath9k_hw
+alias symbol:ath9k_hw_beaconinit ath9k_hw
+alias symbol:genphy_resume libphy
+alias symbol:get_phy_device libphy
+alias symbol:phy_mac_interrupt libphy
+alias symbol:rt2x00mac_remove_interface rt2x00lib
+alias symbol:rt2800_check_firmware rt2800lib
+alias symbol:phy_attach libphy
+alias symbol:rtl92c_phy_set_bb_reg rtl8192c_common
+alias symbol:ath9k_hw_setrxfilter ath9k_hw
+alias symbol:of_phy_connect of_mdio
+alias symbol:rtl92c_phy_set_rfpath_switch rtl8192c_common
+alias symbol:ar9003_mci_send_wlan_channels ath9k_hw
+alias symbol:mdiobus_free libphy
+alias symbol:rtl_cam_reset_all_entry rtlwifi
+alias symbol:rtl_evm_db_to_percentage rtlwifi
+alias symbol:rtl_lps_enter rtlwifi
+alias symbol:ath9k_cmn_beacon_config_sta ath9k_common
+alias symbol:ath_gen_timer_free ath9k_hw
+alias symbol:phy_device_create libphy
+alias symbol:rt2x00mac_start rt2x00lib
+alias symbol:rtl92c_phy_set_io_cmd rtl8192c_common
+alias symbol:rtl92c_dm_check_txpower_tracking rtl8192c_common
+alias symbol:rtl8192_phy_check_is_legal_rfpath rtl8192c_common
+alias symbol:genphy_aneg_done libphy
+alias symbol:rt2x00mac_config rt2x00lib
+alias symbol:cdc_ncm_rx_fixup cdc_ncm
+alias symbol:ath9k_hw_kill_interrupts ath9k_hw
+alias symbol:ath9k_hw_resettxqueue ath9k_hw
+alias symbol:usbnet_open usbnet
+alias symbol:rtl_addr_delay rtlwifi
+alias symbol:mdiobus_register libphy
+alias symbol:phy_register_fixup_for_uid libphy
+alias symbol:rtl92c_dm_init_rate_adaptive_mask rtl8192c_common
+alias symbol:ar9003_mci_send_message ath9k_hw
+alias symbol:rt2x00queue_unmap_skb rt2x00lib
+alias symbol:usbnet_write_cmd_async usbnet
+alias symbol:rt2800_enable_radio rt2800lib
+alias symbol:rtl_tx_mgmt_proc rtlwifi
+alias symbol:ath9k_hw_abort_tx_dma ath9k_hw
+alias symbol:ath9k_hw_btcoex_disable ath9k_hw
+alias symbol:ath9k_hw_rxprocdesc ath9k_hw
+alias symbol:ath_hw_keyreset ath
+alias symbol:of_phy_attach of_mdio
+alias symbol:rtl_query_rxpwrpercentage rtlwifi
+alias symbol:ath9k_hw_stopdmarecv ath9k_hw
+alias symbol:ath9k_hw_disable ath9k_hw
+alias symbol:ath9k_hw_disable_interrupts ath9k_hw
+alias symbol:rt2800_get_tsf rt2800lib
+alias symbol:devm_mdiobus_alloc_size libphy
+alias symbol:rtl92c_phy_sw_chnl rtl8192c_common
+alias symbol:ath_key_delete ath
+alias symbol:rtl_init_rx_config rtlwifi
+alias symbol:usbnet_tx_timeout usbnet
+alias symbol:rt2x00usb_register_read_async rt2x00usb
+alias symbol:genphy_update_link libphy
+alias symbol:ath9k_hw_name ath9k_hw
+alias symbol:read_efuse_byte rtlwifi
+alias symbol:rtl_usb_resume rtl_usb
+alias symbol:ath9k_hw_set_txpowerlimit ath9k_hw
+alias symbol:rtl92c_download_fw rtl8192c_common
+alias symbol:rt2800_disable_radio rt2800lib
+alias symbol:rt2x00queue_start_queues rt2x00lib
+alias symbol:rt2x00mac_set_key rt2x00lib
+alias symbol:ath9k_hw_init_global_settings ath9k_hw
+alias symbol:ath_regd_init ath
+alias symbol:rtl_action_proc rtlwifi
+alias symbol:phy_ethtool_sset libphy
+alias symbol:usbnet_generic_cdc_bind cdc_ether
+alias symbol:ath9k_hw_btcoex_init_mci ath9k_hw
+alias symbol:of_mdiobus_register of_mdio
+alias symbol:rtl92c_phy_query_bb_reg rtl8192c_common
+alias symbol:rtl92c_phy_set_txpower_level rtl8192c_common
+alias symbol:rt2800_reset_tuner rt2800lib
+alias symbol:rtl92c_dm_rf_saving rtl8192c_common
+alias symbol:_rtl92c_phy_dbm_to_txpwr_idx rtl8192c_common
+alias symbol:usbnet_get_endpoints usbnet
+alias symbol:ath9k_hw_setup_statusring ath9k_hw
+alias symbol:rtl92c_phy_set_io rtl8192c_common
+alias symbol:mii_ethtool_sset mii
+alias symbol:rt2800_txdone_entry rt2800lib
+alias symbol:dm_savepowerindex rtl8192c_common
+alias symbol:ath9k_cmn_debug_modal_eeprom ath9k_common
+alias symbol:ath9k_hw_setantenna ath9k_hw
+alias symbol:ath9k_hw_gettxbuf ath9k_hw
+alias symbol:rtl_bb_delay rtlwifi
+alias symbol:cdc_ncm_select_altsetting cdc_ncm
+alias symbol:rt2x00usb_vendor_request rt2x00usb
+alias symbol:ath10k_core_start ath10k_core
+alias symbol:phy_start_interrupts libphy
+alias symbol:usbnet_get_settings usbnet
+alias symbol:ath9k_hw_gen_timer_stop ath9k_hw
+alias symbol:genphy_suspend libphy
+alias symbol:ath_regd_get_band_ctl ath
+alias symbol:rt2800_config_shared_key rt2800lib
+alias symbol:ath9k_hw_set_gpio ath9k_hw
+alias symbol:mii_check_gmii_support mii
+alias symbol:usbnet_status_stop usbnet
+alias symbol:rt2x00usb_disconnect rt2x00usb
+alias symbol:rt2800_link_stats rt2800lib
+alias symbol:ath9k_cmn_setup_ht_cap ath9k_common
+alias symbol:rt2x00lib_dmastart rt2x00lib
+alias symbol:phy_ethtool_get_wol libphy
+alias symbol:ath9k_hw_reset ath9k_hw
+alias symbol:genphy_config_init libphy
+alias symbol:rt2x00queue_pause_queue rt2x00lib
+alias symbol:rt2x00usb_resume rt2x00usb
+alias symbol:ath9k_hw_get_txq_props ath9k_hw
+alias symbol:ath9k_cmn_beacon_config_adhoc ath9k_common
+alias symbol:rt2800_config_intf rt2800lib
+alias symbol:rtl_process_phyinfo rtlwifi
+alias symbol:rt2x00mac_set_antenna rt2x00lib
+alias symbol:usbnet_probe usbnet
+alias symbol:ath9k_hw_process_rxdesc_edma ath9k_hw
+alias symbol:cdc_ncm_unbind cdc_ncm
+alias symbol:ath10k_core_create ath10k_core
+alias symbol:ath_is_mybeacon ath
+alias symbol:rt2x00usb_flush_queue rt2x00usb
+alias symbol:usbnet_unlink_rx_urbs usbnet
+alias symbol:rt2x00queue_stop_queue rt2x00lib
+alias symbol:rtl_deinit_core rtlwifi
+alias symbol:ath9k_cmn_process_rate ath9k_common
+alias symbol:ath9k_hw_stop_dma_queue ath9k_hw
+alias symbol:ath9k_hw_setuptxqueue ath9k_hw
+alias symbol:ath10k_core_register ath10k_core
+alias symbol:ath9k_hw_computetxtime ath9k_hw
+alias symbol:of_phy_find_device of_mdio
+alias symbol:ath10k_core_destroy ath10k_core
+alias symbol:ath10k_info ath10k_core
+alias symbol:ath9k_cmn_update_txpow ath9k_common
+alias symbol:ath9k_cmn_process_rssi ath9k_common
+alias symbol:rt2800_conf_tx rt2800lib
+alias symbol:rt2x00mac_stop rt2x00lib
+alias symbol:ar9003_mci_cleanup ath9k_hw
+alias symbol:rt2800_config rt2800lib
+alias symbol:ath9k_hw_numtxpending ath9k_hw
+alias symbol:phy_attach_direct libphy
+alias symbol:rtl92c_set_fw_joinbss_report_cmd rtl8192c_common
+alias symbol:rtl_deinit_rfkill rtlwifi
+alias symbol:mii_nway_restart mii
+alias symbol:rt2800_read_eeprom_efuse rt2800lib
+alias symbol:rtl92c_dm_init_edca_turbo rtl8192c_common
+alias symbol:phy_driver_register libphy
+alias symbol:ar9003_paprd_enable ath9k_hw
+alias symbol:ath_gen_timer_alloc ath9k_hw
+alias symbol:_rtl92c_phy_calculate_bit_shift rtl8192c_common
+alias symbol:mdiobus_scan libphy
+alias symbol:ath9k_hw_deinit ath9k_hw
+alias symbol:ath9k_hw_startpcureceive ath9k_hw
+alias symbol:phy_disconnect libphy
+alias symbol:rtl_swlps_beacon rtlwifi
+alias symbol:mdiobus_read libphy
+alias symbol:ath9k_hw_init_btcoex_hw ath9k_hw
+alias symbol:phy_stop_interrupts libphy
+alias symbol:phy_connect libphy
+alias symbol:ath9k_hw_abortpcurecv ath9k_hw
+alias symbol:usbnet_change_mtu usbnet
+alias symbol:cdc_ncm_tx_fixup cdc_ncm
+alias symbol:rtl_ips_nic_on rtlwifi
+alias symbol:usbnet_read_cmd usbnet
+alias symbol:ath9k_hw_btcoex_set_concur_txprio ath9k_hw
+alias symbol:ath9k_hw_getnf ath9k_hw
+alias symbol:rtl_cam_mark_invalid rtlwifi
+alias symbol:rt2x00lib_suspend rt2x00lib
+alias symbol:phy_register_fixup_for_id libphy
+alias symbol:phy_write_mmd_indirect libphy
+alias symbol:ath9k_hw_gen_timer_start ath9k_hw
+alias symbol:ath9k_hw_btcoex_enable ath9k_hw
+alias symbol:rt2800_sta_remove rt2800lib
+alias symbol:rt2x00queue_start_queue rt2x00lib
+alias symbol:usbnet_write_cmd_nopm usbnet
+alias symbol:usbnet_stop usbnet
+alias symbol:mdiobus_unregister libphy
+alias symbol:phy_device_register libphy
+alias symbol:rt2x00mac_configure_filter rt2x00lib
+alias symbol:_rtl92c_phy_fw_rf_serial_write rtl8192c_common
+alias symbol:usbnet_read_cmd_nopm usbnet
+alias symbol:ath9k_hw_check_nav ath9k_hw
+alias symbol:ar9003_get_pll_sqsum_dvc ath9k_hw
+alias symbol:mii_link_ok mii
+alias symbol:rtl_p2p_info rtlwifi
+alias symbol:rt2x00mac_sw_scan_start rt2x00lib
+alias symbol:usbnet_device_suggests_idle usbnet
+alias symbol:ath9k_hw_gpio_get ath9k_hw
+alias symbol:rt2x00queue_map_txskb rt2x00lib
+alias symbol:ath9k_cmn_init_crypto ath9k_common
+alias symbol:phy_init_eee libphy
+alias symbol:ar9003_mci_setup ath9k_hw
+alias symbol:usbnet_manage_power usbnet
+alias symbol:rtl_signal_scale_mapping rtlwifi
+alias symbol:rt2x00usb_vendor_request_buff rt2x00usb
+alias symbol:rt2x00lib_beacondone rt2x00lib
+alias symbol:rt2x00lib_rxdone rt2x00lib
+alias symbol:usbnet_link_change usbnet
+alias symbol:rtl_attribute_group rtlwifi
+alias symbol:ath9k_hw_getchan_noise ath9k_hw
+alias symbol:rtl92c_firmware_selfreset rtl8192c_common
+alias symbol:rt2800_gain_calibration rt2800lib
+alias symbol:ar9003_paprd_setup_gain_table ath9k_hw
+alias symbol:rt2800_process_rxwi rt2800lib
+alias symbol:ath9k_hw_gettsf64 ath9k_hw
+alias symbol:ath9k_hw_set_tx_filter ath9k_hw
+alias symbol:phy_init_hw libphy
+alias symbol:ath10k_core_stop ath10k_core
+alias symbol:_rtl92c_phy_rf_serial_read rtl8192c_common
+alias symbol:phy_resume libphy
+alias symbol:ath9k_hw_disable_mib_counters ath9k_hw
+alias symbol:rtl92c_dm_write_dig rtl8192c_common
+alias symbol:rt2800_config_erp rt2800lib
+alias symbol:usbnet_disconnect usbnet
+alias symbol:ath9k_hw_addrxbuf_edma ath9k_hw
+alias symbol:ath9k_cmn_init_channels_rates ath9k_common
+alias symbol:ar9003_mci_state ath9k_hw
+alias symbol:ath9k_cmn_rx_skb_postprocess ath9k_common
+alias symbol:phy_drivers_unregister libphy
+alias symbol:rt2x00lib_pretbtt rt2x00lib
+alias symbol:rt2x00lib_txdone rt2x00lib
+alias symbol:genphy_setup_forced libphy
+alias symbol:rt2800_probe_hw rt2800lib
+alias symbol:rt2x00queue_stop_queues rt2x00lib
+alias symbol:usbnet_status_start usbnet
+alias symbol:ath9k_hw_setuprxdesc ath9k_hw
+alias symbol:rt2x00usb_watchdog rt2x00usb
+alias symbol:ath_is_world_regd ath
+alias symbol:ath9k_hw_btcoex_bt_stomp ath9k_hw
+alias symbol:ath_key_config ath
+alias symbol:usbnet_get_drvinfo usbnet
+alias symbol:rtl_cmd_send_packet rtlwifi
+alias symbol:ath9k_hw_write_associd ath9k_hw
+alias symbol:ar9003_mci_get_next_gpm_offset ath9k_hw
+alias symbol:ath9k_hw_intrpend ath9k_hw
+alias symbol:rtl_cam_add_one_entry rtlwifi
+alias symbol:phy_device_free libphy
+alias symbol:rt2x00queue_flush_queue rt2x00lib
+alias symbol:phy_ethtool_get_eee libphy
+alias symbol:_rtl92c_store_pwrIndex_diffrate_offset rtl8192c_common
+alias symbol:ath9k_hw_btcoex_init_2wire ath9k_hw
+alias symbol:ath9k_hw_phy_disable ath9k_hw
+alias symbol:rtl_btc_status_false rtlwifi
+alias symbol:phy_ethtool_set_eee libphy
+alias symbol:usbnet_cdc_bind cdc_ether
+alias symbol:_rtl92c_phy_fw_rf_serial_read rtl8192c_common
+alias symbol:ath9k_hw_enable_interrupts ath9k_hw
+alias symbol:rtl_tid_to_ac rtlwifi
+alias symbol:mii_ethtool_gset mii
+alias symbol:ath9k_hw_getrxfilter ath9k_hw
+alias symbol:ath_gen_timer_isr ath9k_hw
+alias symbol:rtl_is_special_data rtlwifi
+alias symbol:rt2x00queue_get_entry rt2x00lib
+alias symbol:ath9k_cmn_rx_accept ath9k_common
+alias symbol:rt2800_rfkill_poll rt2800lib
+alias symbol:usbnet_cdc_status cdc_ether
+alias symbol:ath9k_hw_bstuck_nfcal ath9k_hw
+alias symbol:ath9k_hw_putrxbuf ath9k_hw
+alias symbol:_rtl92c_phy_rf_serial_write rtl8192c_common
+alias symbol:ar9003_paprd_create_curve ath9k_hw
+alias symbol:phy_driver_unregister libphy
+alias symbol:rtlwifi_rate_mapping rtlwifi
+alias symbol:ath10k_core_unregister ath10k_core
+alias symbol:usbnet_defer_kevent usbnet
+alias symbol:ath10k_print_driver_info ath10k_core
+alias symbol:rtl_ops rtlwifi
+alias symbol:rtl_rfreg_delay rtlwifi
+alias symbol:ath9k_hw_btcoex_set_weight ath9k_hw
+alias symbol:rt2800_load_firmware rt2800lib
+alias symbol:phy_stop libphy
+alias symbol:cdc_ncm_rx_verify_nth16 cdc_ncm
+alias symbol:phy_print_status libphy
+alias symbol:rtl_send_smps_action rtlwifi
+alias symbol:rt2x00mac_flush rt2x00lib
+alias symbol:ath9k_hw_cfg_output ath9k_hw
+alias symbol:rtl92c_phy_rf_config rtl8192c_common
+alias symbol:rt2800_link_tuner rt2800lib
+alias symbol:phy_find_first libphy
+alias symbol:rtl_lps_leave rtlwifi
+alias symbol:_rtl92c_phy_txpwr_idx_to_dbm rtl8192c_common
+alias symbol:rt2800_get_tkip_seq rt2800lib
+alias symbol:usbnet_resume_rx usbnet
+alias symbol:rt2x00mac_sta_add rt2x00lib
+alias symbol:phy_drivers_register libphy
+alias symbol:rtl92c_set_fw_rsvdpagepkt rtl8192c_common
+alias symbol:rtl_fw_cb rtlwifi
+alias symbol:usbnet_get_link usbnet
+alias symbol:cdc_ncm_bind_common cdc_ncm
+alias symbol:ath9k_hw_setrxabort ath9k_hw
+alias symbol:ath9k_hw_txstart ath9k_hw
+alias symbol:ath9k_hw_btcoex_init_scheme ath9k_hw
+alias symbol:rtl_cam_get_free_entry rtlwifi
+alias symbol:rt2x00usb_clear_entry rt2x00usb
+alias symbol:rtl92c_fill_h2c_cmd rtl8192c_common
+alias symbol:genphy_restart_aneg libphy
+alias symbol:_rtl92c_phy_init_bb_rf_register_definition rtl8192c_common
+alias symbol:ath9k_hw_setopmode ath9k_hw
+alias symbol:rtl92c_dm_bt_coexist rtl8192c_common
+alias symbol:rt2800_set_rts_threshold rt2800lib
+alias symbol:rt2x00queue_unpause_queue rt2x00lib
+alias symbol:rt2800_get_txwi_rxwi_size rt2800lib
+alias symbol:rt2x00usb_vendor_req_buff_lock rt2x00usb
+alias symbol:rt2x00usb_kick_queue rt2x00usb
+alias symbol:rtl92c_set_p2p_ps_offload_cmd rtl8192c_common
+alias symbol:rtl92c_bt_rssi_state_change rtl8192c_common
+alias symbol:ath9k_cmn_reload_chainmask ath9k_common
+alias symbol:efuse_read_1byte rtlwifi
+alias symbol:rt2x00lib_txdone_noinfo rt2x00lib
+alias symbol:rt2800_clear_beacon rt2800lib
+alias symbol:rt2x00mac_tx_frames_pending rt2x00lib
+alias symbol:ath9k_hw_btcoex_init_3wire ath9k_hw
+alias symbol:phy_start libphy
+alias symbol:phy_read_mmd_indirect libphy
+alias symbol:phy_register_fixup libphy
+alias symbol:ath9k_hw_set_rx_bufsize ath9k_hw
+alias symbol:ath_rxbuf_alloc ath
+alias symbol:ath9k_hw_reset_calvalid ath9k_hw
+alias symbol:phy_connect_direct libphy
+alias symbol:rtl_lps_change_work_callback rtlwifi
+alias symbol:rtl_ps_enable_nic rtlwifi
+alias symbol:rtl92c_phy_update_txpower_dbm rtl8192c_common
+alias symbol:usbnet_purge_paused_rxq usbnet
+alias symbol:rt2x00mac_bss_info_changed rt2x00lib
+alias symbol:ath9k_hw_wait ath9k_hw
+alias symbol:usbnet_suspend usbnet
+alias symbol:ath9k_hw_cfg_gpio_input ath9k_hw
+alias symbol:rtl_hal_pwrseqcmdparsing rtlwifi
+alias symbol:mdiobus_alloc_size libphy
+alias symbol:ath9k_cmn_get_hw_crypto_keytype ath9k_common
+alias symbol:rt2800_config_ant rt2800lib
+alias symbol:ath9k_hw_releasetxqueue ath9k_hw
+alias symbol:ath9k_hw_set_txq_props ath9k_hw
+alias symbol:rt2800_disable_wpdma rt2800lib
+alias symbol:rt2800_vco_calibration rt2800lib
+alias symbol:phy_suspend libphy
+alias symbol:phy_start_aneg libphy
+alias symbol:generic_mii_ioctl mii
+alias symbol:usbnet_update_max_qlen usbnet
+alias symbol:rtl_ps_disable_nic rtlwifi
+alias symbol:rtl_usb_disconnect rtl_usb
+alias symbol:ath9k_cmn_debug_stat_rx ath9k_common
+alias symbol:ath_printk ath
+alias symbol:mii_check_media mii
+alias symbol:rt2x00mac_tx rt2x00lib
+alias symbol:rtl92c_phy_sw_chnl_callback rtl8192c_common
+alias symbol:ath9k_hw_settsf64 ath9k_hw
+alias symbol:mii_check_link mii
+alias symbol:ath9k_hw_setmcastfilter ath9k_hw
+alias symbol:efuse_one_byte_read rtlwifi
+alias symbol:rt2x00usb_regbusy_read rt2x00usb
+alias symbol:usbnet_nway_reset usbnet
+alias symbol:ath10k_err ath10k_core
+alias symbol:rtl_usb_probe rtl_usb
+alias symbol:ath9k_cmn_beacon_config_ap ath9k_common
+alias symbol:ath_hw_get_listen_time ath
+alias symbol:usbnet_skb_return usbnet
+alias symbol:ath_hw_cycle_counters_update ath
+alias symbol:rtl92c_dm_init rtl8192c_common
+alias symbol:rt2x00lib_get_bssidx rt2x00lib
+alias symbol:rtl92ce_phy_set_rf_on rtl8192c_common
+alias symbol:rtl92c_set_fw_pwrmode_cmd rtl8192c_common
+alias symbol:rtl_usb_suspend rtl_usb
+alias symbol:rt2x00mac_sw_scan_complete rt2x00lib
+alias symbol:ar9003_mci_get_interrupt ath9k_hw
+alias symbol:phy_detach libphy
+alias symbol:usbnet_cdc_unbind cdc_ether
+alias symbol:rt2x00usb_disable_radio rt2x00usb
+alias symbol:rtl_get_tcb_desc rtlwifi
+alias symbol:usbnet_start_xmit usbnet
+alias symbol:rt2x00mac_sta_remove rt2x00lib
+alias symbol:rtl_evm_dbm_jaguar rtlwifi
+alias symbol:rt2x00usb_uninitialize rt2x00usb
+alias symbol:rt2800_ampdu_action rt2800lib
+alias symbol:usbnet_set_settings usbnet
+alias symbol:rtl_cam_empty_entry rtlwifi
+alias symbol:rt2x00lib_resume rt2x00lib
+alias symbol:cdc_ncm_rx_verify_ndp16 cdc_ncm
+alias symbol:rt2x00queue_flush_queues rt2x00lib
+alias symbol:rt2x00mac_add_interface rt2x00lib
+alias symbol:ath9k_hw_init ath9k_hw
+alias symbol:dm_writepowerindex rtl8192c_common
+alias symbol:rt2x00usb_initialize rt2x00usb
+alias symbol:rtl_cam_del_entry rtlwifi
+alias symbol:rt2x00lib_remove_dev rt2x00lib
+alias symbol:rtl92c_phy_lc_calibrate rtl8192c_common
